Mathlib/FieldTheory/SplittingField/IsSplittingField.lean

```diff
@@ -121,6 +128,12 @@ theorem finiteDimensional (f : K[X]) [IsSplittingField K L f] : FiniteDimensiona
       if hf : f = 0 then by rw [hf, rootSet_zero] at hy; cases hy
       else IsAlgebraic.isIntegral ⟨f, hf, (mem_rootSet'.mp hy).2⟩⟩
 
+theorem IsScalarTower.isAlgebraic [Algebra F K] [Algebra F L] [Algebra.IsAlgebraic F K]
+    [IsScalarTower F K L] (x : K) [IsSplittingField K L (mapAlg F K (minpoly F x))] :
+    Algebra.IsAlgebraic F L := by
+  let _ : FiniteDimensional K L := IsSplittingField.finiteDimensional _ (mapAlg F K (minpoly F x))
+  exact Algebra.IsAlgebraic.trans F K L
```

Comment:
This one can also be generalized.
```suggestion
theorem IsScalarTower.isAlgebraic [Algebra F K] [Algebra F L] [Algebra.IsAlgebraic F K]
    [IsScalarTower F K L] (f : K[X]) [IsSplittingField K L f] :
    Algebra.IsAlgebraic F L := by
  have : FiniteDimensional K L := IsSplittingField.finiteDimensional L f
  exact Algebra.IsAlgebraic.trans F K L
```

Reply:
(I'm not sold on the utility of this one, since it's basically just "splitting field of algebraic extension is algebraic" but there are other combinations like "algebraic extension of splitting field is algebraic", "splitting field of splitting field is algebraic").